easybuild/easyconfigs/n/ncbi-vdb/ncbi-vdb-2.5.8-1-foss-2016b.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'ncbi-vdb'
+version = '2.5.8-1'
+
+homepage = 'https://github.com/ncbi/ncbi-vdb'
+description = """The SRA Toolkit and SDK from NCBI is a collection of tools and libraries for
+ using data in the INSDC Sequence Read Archives."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://github.com/ncbi/ncbi-vdb/archive/']
+sources = ['%(version)s.tar.gz']
+
+dependencies = [
+    ('libxml2', '2.9.4'),
+    ('file', '5.28'),  # provides libmagic
+    ('HDF5', '1.8.17'),
```

Comment:
there's an HDF5 `1.10.0-patch1` release, which is backwards compatible (I think), but not sure it's a good idea to pick that up as a dependency already...